app/src/main/java/org/jellyfin/mobile/bridge/ExternalPlayer.kt

```diff
@@ -42,6 +43,13 @@ class ExternalPlayer(private val fragment: WebViewFragment) : KoinComponent {
                     putExtra("position", mediaSource?.mediaStartMs)
                     putExtra("return_result", true)
                     putExtra("secure_uri", true)
+                    val selectedTrack = mediaSource?.subtitleTracksGroup?.tracks?.getOrNull(mediaSource?.subtitleTracksGroup?.selectedTrack as Int)?.url
```

Comment:
Instead of using so many null-safe accessors, couldn't you make `mediaSource` a local non-null variable?

Reply:
I'm not sure if I should, mediaSource is originally a JSON object and I can't be sure that it will always have the necessary properties.

Reply:
That's checked internally of JellyfinMediaSource, so no worries there. And the mediaSource itself still doesn't need to be a nullable var.

Reply:
Roger that.